raiden_contracts/data/gas.json

```diff
@@ -3,21 +3,21 @@
     "CustomToken.mint": 49423,
     "CustomToken.transfer": 36123,
     "CustomToken.transferFrom": 29557,
-    "MonitoringService.claimReward": 41942,
-    "MonitoringService.monitor": 207730,
-    "OneToN.claim": 92786,
+    "MonitoringService.claimReward": 41910,
+    "MonitoringService.monitor": 207666,
+    "OneToN.claim": 92769,
     "SecretRegistry.registerSecret": 46892,
-    "ServiceRegistry.deposit": 502020,
-    "ServiceRegistry.setURL": 44589,
+    "ServiceRegistry.deposit": 527654,
+    "ServiceRegistry.setURL": 44678,
```

Comment:
ServiceRegistry has real changes, but the other numbers are changing without any known reasons.  If anybody figures out why, document in https://github.com/raiden-network/raiden-contracts/issues/910